flag_engine/identities/traits/models.py

```diff
@@ -1,8 +1,8 @@
-import typing
-from dataclasses import dataclass
+from pydantic import BaseModel
 
+from flag_engine.identities.traits.types import TraitValue
 
-@dataclass
-class TraitModel:
+
+class TraitModel(BaseModel):
     trait_key: str
-    trait_value: typing.Any
+    trait_value: TraitValue = ...
```

Comment:
What does the `...` signify here? Why not use `None`?

Reply:
`...` signifies a required `__init__` argument. Even when we're setting the value to `None`, we're required to do this explicitly. This was implemented in the Marshmallow schema too.